README.md

```diff
@@ -95,6 +95,12 @@ npm install nanostores
 * [Logux Client](https://github.com/logux/client): stores with WebSocket
   sync and CRDT conflict resolution.
 
+## Devtools
```

Comment:
1. Can you also add `Devtools` to TOC?
2. I know that it is strange code style, but can you put double `\n` before the headers (it emulates bigger margin)